HandsOn/Group07/rdf/EventoDeportivo.ttl

```diff
@@ -0,0 +1,57 @@
+@base <http://www.semanticweb.org/> .
+@prefix datosbne: <http://www.semanticweb.org/instances/> .
```

Comment:
This one is not finally used, I assume. In any case, there are no problems with this one.